utils/swift_build_support/swift_build_support/targets.py

```diff
@@ -8,8 +8,11 @@
 # See https://swift.org/LICENSE.txt for license information
 # See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
+import atexit
```

Comment:
I might be missing something, but where are these imports being used?

Reply:
That isn't part of 8f4a746, and I'm not actually certain what added it — but in any case the Python code gets linted so unused imports have to be removed for the tests to pass (I know I've removed a few of these upstream).